lib/engines/hcktest/hcktest.rb

```diff
@@ -295,8 +302,8 @@ def group_tests_by_config
       grouped_tests
     end
 
-    def run
-      upload_driver_package unless @driver_path.nil?
+    def run # rubocop:disable Metrics/AbcSize
```

Comment:
Refactor rather than disabling RuboCop.
In this case, I think you can remove the message "AutoHCK started in dump only mode"; it is obvious and we are not writing such a message for the auto run anyway.